### DISCUSSION TITLE
show_arrows in manually created gpx file isn't shown

### DISCUSSION THREAD

**Discussion by kosi2711** [osmandapp/OsmAnd#15077] [Q&A]
Hello alltogether,

I collect all gpx files from my hiking guides in a single gpx file and assign each \<wpt\>/\<trk\> a different color and icon. Here is an abstract:

```
...
<wpt lat="44.424078008159995" lon="8.93770138733089">
  <name>12.671 Km, 278/672 m, 0.000 Km CinTe_01_Righi-Forte_Diamante.gpx</name>
  <desc>12.671 Km, 278/672 m, 0.000 Km, filename=../../tracks-01/z14-Rother-Cinque-Terre/Rother-Touren/CinTe_01_Righi-Forte_Diamante.gpx</desc>
  <extensions>
    <osmand:color>#50ffff00</osmand:color>
    <osmand:icon>special_trekking</osmand:icon>
    <osmand:background>square</osmand:background>
  </extensions>
</wpt>
.... more <wpt>'s ...
  <trk>
  <name>CinTe_01_Righi-Forte_Diamante.gpx</name>
  <extensions>
    <osmand:color>#50ffff00</osmand:color>
    <osmand:show_arrows>true</osmand:show_arrows>
  </extensions>
    <trkseg>
<trkpt lat="44.424078008159995" lon="8.93770138733089">
... all <trkpt>s 
</trk>
<trk>
....
```

Everything works with the exception of "show_arrows". These are not displayed - I have to do it manually (I've tried putting show_arrows in the wpt section, but this also doesn't work).

Q1: Isn't "show_arrows" evaluated?
Q2: can I assign different colors to the arrows?

Greets and thx in advance!

**Comment by vshcherb** [comment]
It should be part of <gpx> extensions - https://docs.osmand.net/docs/technical/osmand-file-formats/osmand-gpx

**Reply by kosi2711** [reply]
thanks for your quick response - my mistake was that I've added "show_arrows" only under the \<extensions\> of \<wpt\>, but not in the \<gpx\>-section.

But all of them get the same color. Is it possible to assign different colors for the arrows (the same color as the one from the \<wpt\> or \<trk\>?
